lib/srv/termmanager.go

```diff
@@ -194,7 +174,7 @@ func (g *TermManager) On() {
 	g.mu.Lock()
 	defer g.mu.Unlock()
 	g.on = true
-	g.readStateUpdate.Broadcast()
+	g.readStateUpdate <- true
```

Comment:
An unfortunate side-effect of the change is that this could now deadlock if `Read` is not running and we get multiple calls to `On()` / `Off()`.

Since the channel doesn't really need to send information over, just a signal, I think we could use a select and be alright.